apis/sesv2/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,9 @@
+ignore:
```

Comment:
ImportJobSummary, BlacklistEntry, Contact, DailyVolume, CustomVerificationEmailTemplateMetadata - I notice a few things hand-edited in the generated files, could you add them here?

Reply:
Thanks @chlunde for highlighting, Those structs are not hand-edited by me, I notice that is it auto-generated probably because i didn't add the mentioned list in the `ignore:` items. Once we fixed the `check-diff` issue below, I will push all the changes again.

Reply:
Hi @chlunde , I have added the list of `resources` and `field_paths` to resolve unused one